src/corelibs/U2Formats/src/TextDocumentFormat.cpp

```diff
@@ -22,23 +22,24 @@
 #include "TextDocumentFormat.h"
 
 #include <U2Core/IOAdapter.h>
+#include <U2Core/IOAdapterTextStream.h>
 #include <U2Core/TextUtils.h>
 #include <U2Core/U2OpStatus.h>
 
 namespace U2 {
 
-TextDocumentFormat::TextDocumentFormat(QObject *p, const DocumentFormatId &id, DocumentFormatFlags _flags, const QStringList &fileExts)
+TextDocumentFormatDeprecated::TextDocumentFormatDeprecated(QObject *p, const DocumentFormatId &id, DocumentFormatFlags _flags, const QStringList &fileExts)
```

Comment:
I don't get why this class is called deprecated now. Are we about to replace it? As far as I understand, we just switched the adapter